src/pycel/excellib.py

```diff
@@ -144,11 +144,10 @@ def count(*args):
                if isinstance(x, (int, float)) and not isinstance(x, bool))
 
 
+@excel_helper(err_str_params=None, array_params=0)
```

Comment:
I've set `err_str_params=None` to keep existing error behavior

Reply:
actually `err_str_params=None` is needed because in the case of `countif` excel allows error code (such as #REF) to be passed in as `range` or `criteria` ; I've added some tests to describe the behavior